app/models/cms/template.rb

```diff
@@ -10,6 +12,22 @@ class CMS::Template < ApplicationRecord
   scope :but, ->(*klasses) { where{ type.not_in klasses.map(&:to_s) } }
   scope :recents, -> { order(updated_at: :desc).where.has { updated_at != created_at } }
 
+  self.allowed_search_scopes = %i[type section_id]
+
+  scope :by_type, ->(type) {
+    type_class = {
```

Comment:
I'm wondering whether this mapping could be added as constant, in case it might be useful in some other place. Although I guess we shouldn't just add it and not use anywhere else (or maybe we could :upside_down_face:)

Reply:
Now it's using CMS::TypeMap. So it's much simpler.